cockatrice/src/carddatabase.cpp

```diff
@@ -318,107 +378,151 @@ const QChar CardInfo::getColorChar() const
     {
         case 0:
             return QChar();
-            break;
         case 1:
             return colors.first().isEmpty() ? QChar() : colors.first().at(0);
-            break;
         default:
             return QChar('m');
-            break;
     }
 }
 
 static QXmlStreamWriter &operator<<(QXmlStreamWriter &xml, const CardInfo *info)
 {
+    if (! info)
+    {
+        qDebug() << "operator<< (~376) info is nullptr";
```

Comment:
What is 376?

Reply:
Was referring to the line number, i'll be more explicit